FxCopRules.ruleset

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<RuleSet Name="GitExtensions Custom Rules" Description="This rule set contains all rules. Running this rule set may result in a large number of warnings being reported. Use this rule set to get a comprehensive picture of all issues in your code. This can help you decide which of the more focused rule sets are most appropriate to run for your projects." ToolsVersion="11.0">
+<RuleSet Name="GitExtensions FxCop Rules" Description="This rule set contains all rules. Running this rule set may result in a large number of warnings being reported. Use this rule set to get a comprehensive picture of all issues in your code. This can help you decide which of the more focused rule sets are most appropriate to run for your projects." ToolsVersion="11.0">
```

Comment:
Re-labelled this RuleSet for clarity. I'm not sure it's needed any more.

Reply:
💭 Seems like this file can be removed since you copied the configuration to the new ruleset.

Reply:
If not then we can probably remove `FxCop.targets` too.